client/asset/interface.go

```diff
@@ -130,10 +130,25 @@ type ConfigOption struct {
 	DisplayName  string      `json:"displayname"`
 	Description  string      `json:"description"`
 	DefaultValue interface{} `json:"default"`
-	NoEcho       bool        `json:"noecho"`
-	IsBoolean    bool        `json:"isboolean"`
+	// If MaxValue/MinValue are set to the string "now" for a date config, the
+	// UI will display the current date.
+	MaxValue          interface{} `json:"max"`
+	MinValue          interface{} `json:"min"`
+	NoEcho            bool        `json:"noecho"`
+	IsBoolean         bool        `json:"isboolean"`
+	IsDate            bool        `json:"isdate"`
+	DisableWhenActive bool        `json:"disablewhenactive"`
+	IsBirthdayConfig  bool        `json:"isBirthdayConfig"`
```

Comment:
The behavior I wanted on the UI was to have the default value be "now" if the current installation created the seed, but otherwise be the default value passed in the config.